Updating to `apollo-angular@3.0.0` produces  `error TS2307: Cannot find module 'apollo-angular/types' or its corresponding type declarations.`

**Describe the bug**
Release 3.0.0 introduces an undocumented breaking change by moving the import location of types from `apollo-angular/types` to `apollo-angular/build/types` or `apollo-angular/src/types`.

**To Reproduce**
Update an `angular-apollo@2.x.x` project importing types, such as `import { QueryOptionsAlone, SubscriptionOptionsAlone, WatchQueryOptionsAlone } from 'apollo-angular/types';`.

**Expected behavior**
The import location should not change or [the Migration Guide](https://apollo-angular.com/docs/migration) should specify the new import location.

**Environment:**
```
├── @angular/cli@13.2.4 
├── UNMET PEER DEPENDENCY @angular/core@13.2.3 
├── @apollo/client@3.5.9 
├── @types/graphql@14.5.0
├── apollo-angular@3.0.0 
├── barrel-maintainer@1.5.3
├── graphql@16.3.0 
└── typescript@4.5.5 
```

**Additional context**
Thanks for the new release!

I confirm this problem.  Passing from v3.0.0-alpha.2 to v3 I lost some imports
```
Cannot find module 'apollo-angular/types' or its corresponding type declarations.
import { MutationResult } from 'apollo-angular/types';
```

--- a/.changeset/curly-rivers-complain.md
+++ b/.changeset/curly-rivers-complain.md
@@ -1,0 +1,6 @@
+---
+'apollo-angular': patch
+---
+
+Export MutationOptionsAlone, QueryOptionsAlone, SubscriptionOptionsAlone, WatchQueryOptionsAlone
+from 'apollo-angular'

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -152,6 +152,7 @@ jobs:
 
       - name: Setup E2E tests
         run: |
+          sudo apt-get update
           sudo apt-get install libgtk2.0-0t64 libgtk-3-0t64 libgbm-dev libnotify-dev libnss3 libxss1 libasound2t64 libxtst6 xauth xvfb
           (cd testapp && yarn add -D cypress-fail-on-console-error)
 

--- a/packages/apollo-angular/src/index.ts
+++ b/packages/apollo-angular/src/index.ts
@@ -7,13 +7,17 @@ export { Mutation } from './mutation';
 export { Subscription } from './subscription';
 export { APOLLO_OPTIONS, APOLLO_NAMED_OPTIONS, APOLLO_FLAGS } from './tokens';
 export type {
-  NamedOptions,
-  SubscriptionResult,
-  WatchQueryOptions,
   ExtraSubscriptionOptions,
-  MutationResult,
   Flags,
-  VariablesOf,
+  MutationOptionsAlone,
+  MutationResult,
+  NamedOptions,
+  QueryOptionsAlone,
   ResultOf,
+  SubscriptionOptionsAlone,
+  SubscriptionResult,
+  VariablesOf,
+  WatchQueryOptions,
+  WatchQueryOptionsAlone,
 } from './types';
 export { gql, graphql } from './gql';

